code/modules/mob/living/simple_animal/bot/firebot.dm

```diff
@@ -103,12 +103,12 @@
 	last_found = world.time
 	update_appearance()
 
-/mob/living/simple_animal/bot/firebot/emag_act(mob/user)
-	..()
+/mob/living/simple_animal/bot/firebot/emag_act(mob/user, obj/item/card/emag/emag_card)
+	. = ..()
 	if(!(bot_cover_flags & BOT_COVER_EMAGGED))
 		return
 	if(user)
-		to_chat(user, span_danger("[src] buzzes and beeps."))
+		balloon_alert(user, "indiscriminate fighting with fire mode engaged") //hehe. funny
```

Comment:
This is a funnier message but it's a bit long for a balloon alert 😔

Reply:
very well. a to chat it is